src/seedu/addressbook/parser/Parser.java

```diff
@@ -34,6 +35,9 @@
     public static final Pattern KEYWORDS_ARGS_FORMAT =
             Pattern.compile("(?<keywords>\\S+(?:\\s+\\S+)*)"); // one or more keywords separated by whitespace
 
+    public static final Pattern PHONE_NUMBER_ARGS_FORMAT =
+            Pattern.compile("(?<phoneNumbers>\\S+(?:\\s+\\S+)*)"); // one or more phone numbers separated by whitespace
```

Comment:
Is it necessary to create a new format here? Can we just use the `KEYWORDS_ARGS_FORMAT`? Does that mean phone number is not a keyword argument?